lib/mongo/retryable.rb

```diff
@@ -548,5 +496,51 @@ def raise_unsupported_error(e)
       new_error.set_backtrace(e.backtrace)
       raise new_error
     end
+
+    def retryable_exceptions
```

Comment:
We try to add docstring to all methods. For private methods the docstring do not have to be long and detailed as for public ones, but a short description would be nice.

Reply:
Good to know! I saw many of the private methods in that file didn't have docstrings, and tried to follow that pattern. I was obviously led astray. :) I'll get that fixed.